optuna/integration/allennlp/_executor.py

```diff
@@ -212,6 +210,19 @@ def run(self) -> float:
 
         self._set_environment_variables()
         params = allennlp.common.params.Params(self._build_params())
+
+        if "distributed" in params:
+
+            if "OPTUNA_ALLENNLP_USE_DISTRIBUTED" in os.environ:
+                warnings.warn(
+                    "Other process may already exists."
+                    " If you have some trouble, please unsed an environment"
+                    " variable `OPTUNA_ALLENNLP_USE_DISTRIBUTED`"
+                    " and try it again."
+                )
+
+            os.environ["OPTUNA_ALLENNLP_USE_DISTRIBUTED"] = "1"
```

Comment:
One global variable is set to let a pruner know if it is initialized in distributed scenario.